composer.json

```diff
@@ -101,7 +101,7 @@
     "conflict": {
         "phpdocumentor/reflection-docblock": "<3.0||>=3.2.0,<3.2.2",
         "phpdocumentor/type-resolver": "<0.2.1",
-        "phpunit/phpunit": "<4.8.35|<5.4.3,>=5.0"
+        "phpunit/phpunit": "<5.4.3"
```

Comment:
Why did you remove the `,>=5.0`?

Reply:
Lower bound constraint is not needed anymore